src/content/docs/en/pages/deploy-journey/automate/zero-touch-provisioning/zero-touch-provisioning.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Automate with zero-touch provisioning
+description: Automate the deployment on premises, multi-cloud and for IoT devices through Azion Edge Orchestrator.
+meta_tags: deploy, orchestrator, provisioning, ztp
+namespace: docs_deploy_automate_zero_touch_provisioning
+menu_namespace: deployMenu
+permalink: /documentation/products/deploy/automate/zero-touch-provisioning/
+---
+
+import Button from '~/components/Button.astro'
+
+[Azion Edge Orchestrator](/en/documentation/products/deploy/edge-orchestrator/) is a programmable and extensible cloud-management and *zero-touch provisioning* orchestration service purposely built for edge infrastructures and applications.
+
+<Button href="/en/documentation/products/guides/deploy/install-orchestrator-agent/" text="go to how to install edge orchestrator agent" variant="secondary" target="_blank"></Button>
+<br />
+<Button href="/en/documentation/products/guides/deploy/authorize-an-edge-node/" text="go to how to authorize an edge node" variant="secondary" target="_blank"></Button>
+
+## Zero-touch Provisionig
+
+Zero-touch provisioning is a feature that allows you to automate file and configuration provisioning.
+
+Previously, the deployment of a device and all its access points was a multi-step process that required involvement from multiple professionals. With the introduction of zero-touch provisioning technology, this process has been streamlined, accelerating deployment times and enhancing security.
+
+Basic workflow for using zero-touch provisioning through Azion Edge Orchestrator:
+
+1. Install the Edge Orchestrator Agent on your device. This device now represents an edge node.
+2. Authorize the node on the Azion platform.
+3. The node is now ready to be provisioned automatically.
+
+### Deploy on premises
+
+Zero-touch provisioning can significantly benefit on-premises nodes in several ways:
+
+1. **Automated Provisioning**: it allows for the automatic provisioning of configurations, which can significantly reduce the time and effort required to set up each node.
+
+2. **Enhanced Security**: by eliminating the need for manual configuration, it reduces the risk of human error, which can lead to security vulnerabilities.
+
+3. **Scalability**: it makes it easier to scale your network, as new nodes can be added and configured automatically.
+
+4. **Efficiency**: it streamlines the deployment process, accelerating deployment times and reducing the need for involvement from multiple professionals.
+
+### Deploy on multi-cloud 
+
+Zero-touch provisioning can significantly benefit multi-cloud environments:
+
+1. **Automated Provisioning**: in a multi-cloud environment, managing configurations across different cloud platforms can be complex. It can automate this process, reducing the time and effort required.
```

Comment:
```suggestion
1. **Automated provisioning**: in a multi-cloud environment, managing configurations across different cloud platforms can be complex. It can automate this process, reducing the time and effort required.
```